connectors/elastic-connector/elasticConnectorModel.go

```diff
@@ -10,10 +10,20 @@ import (
 	"github.com/gwos/tcg/transit"
 	"strings"
 	"sync"
+	"time"
 )
 
 var doOnce sync.Once
 
+var initStatusMessages = map[transit.MonitorStatus]string{
```

Comment:
Make sure we have all standard statuses listed here. What about non-match?

Reply:
If message for status not found service's status text will be only thresholds text if they exist, empty if thresholds do not exist